### DISCUSSION TITLE
MVPS HOSTS Filter List - Questions / Discussion

### DISCUSSION THREAD

**Discussion by ghost** [uBlockOrigin/uBlock-issues#2005] [General]
This filter list has had no updates since March-06-2021 and, by default, is not enabled.

Here are my questions:

1.) How useful is it really with it being that out-of-date?

2.) Is it worth keeping it included with uBO?

**Comment by uBlock-user** [comment]
Author mentioned his illness in March 2021 - https://winhelp2002.mvps.org/hosts.htm

**Comment by Yuki2718** [comment]
We applied 1 year rule for removal of a malware list, and it's almost 1 year since last MVPS update. Probably the list should be removed.

**Comment by bandaid** [comment]
no, the mvps list is still valid. even valid after a picallion years from now.

**Comment by ghost** [comment]
List removed: https://github.com/gorhill/uBlock/commit/858fdaced3fbaf2d4c16aaa71cdab2b050e890b7

**Reply by ghost** [reply]
Thank you kindly! 😃
